src/res/translation/translation_zh_CN.ts

```diff
@@ -2007,7 +2007,7 @@ E&apos; stato disattivato l&apos;audio del tuo canale ed inserito il &quot;Disat
     <message>
         <location filename="../../clientsettingsdlgbase.ui" line="1046"/>
         <source>Feedback Protection</source>
-        <translation>防止声音正反馈循环（啸叫）</translation>
+        <translation type="unfinished">自激保护</translation>
```

Comment:
I assume this needs to be approved?

Reply:
Oh I realized this issue (it marked as unfinished) after I push my last commit, I must wrongly clicked some button from Qt Linguist program. Will fix that in my next commit. My apartment got power failure due to a healvy downpour so I cannot use my computer today, sorry about that.

Reply:
No hurry ;-).